src/item.cpp

```diff
@@ -2298,6 +2245,70 @@ std::string item::display_name(unsigned int quantity) const
     return string_format("%s%s%s", name.c_str(), side.c_str(), qty.c_str());
 }
 
+std::string item::get_item_tags(bool parens) const
+{
+    std::string rtn;
+    std::stringstream ret;
+    ret.str("");
+    
+    if (is_food()) {
+        if(rotten()) {
+            ret << _(parens ? " (rotten)" : " ROTTEN");
```

Comment:
```_()``` marks strings for translation, so you need to modify this line to embrace only string ```" (rotten)"```